db.json

```diff
@@ -6369,6 +6373,10 @@
     "compressible": true,
     "extensions": ["vtt"]
   },
+  "text/x-ansi": {
+    "compressible": true,
+    "extensions": ["ans"]
+  },
```

Comment:
Can't seem to find any original source what-so-ever for this one for MIME/Content-Type but I have seen these in archive files back in the day. Typically just about any file extension *(recall `.nfo` mainly back in my BBS days with X/Y/Z-Modem)* could have 8-bit ANSI escaped sequences in them. 7-bit is pretty bland but can have ascii art in them too. Mac vs IBM/PC is also a factor with ANSI since it wasn't industry standard and rivals. Thus the birth of Unicode came along... too unify character sets. Anyhow... history lesson from my perspective complete. ;)

The real question I think is... is there software out there that needs a MIME type for these... if so then probably the developer version of `text/x-ansi` versus a production/release/golden/official is a good bet.